src/org/omegat/filters2/master/PluginUtils.java

```diff
@@ -100,6 +100,7 @@ public static void loadPlugins(final Map<String, String> params) {
             URL[] urls = new URL[fs.size()];
             for (int i = 0; i < urls.length; i++) {
                 urls[i] = fs.get(i).toURI().toURL();
+                THEME_PLUGIN_JARS.add(urls[i]); // FIXME: only required files should be added
```

Comment:
What is your plan for fixing this?

Reply:
I have not had clear idea yet.
I tried do this in L.118 at first, but `URL mu` is an URL of `MENIFEST.MF` not jar file.
So here is too late to do thing.
```
                    if ("theme".equals(m.getMainAttributes().getValue("Plugin-Category"))) {
                        // FIXME: add jar file to list
                     }
```

When I found a way to detect plugins which need to be in the System Class-Path, it can be fixed.